droid-swing-ui/src/main/java/uk/gov/nationalarchives/droid/gui/export/ExportDialog.java

```diff
@@ -402,24 +417,28 @@ public void actionPerformed(ActionEvent evt) {
             }
         });
 
+        buttonGroup2.add(RadioCSVOutput);
+        RadioCSVOutput.setSelected(true);
+        RadioCSVOutput.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioCSVOutput.text")); // NOI18N
+        RadioCSVOutput.setToolTipText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioCSVOutput.toolTipText")); // NOI18N
+
+        buttonGroup2.add(RadioJSONOutput);
+        RadioJSONOutput.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioJSONOutput.text")); // NOI18N
+        RadioJSONOutput.setToolTipText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioJSONOutput.toolTipText")); // NOI18N
+
         GroupLayout jPanelBottomControlLayout = new GroupLayout(jPanelBottomControl);
         jPanelBottomControl.setLayout(jPanelBottomControlLayout);
         jPanelBottomControlLayout.setHorizontalGroup(jPanelBottomControlLayout.createParallelGroup(Alignment.LEADING)
             .addGroup(jPanelBottomControlLayout.createSequentialGroup()
                 .addContainerGap()
                 .addGroup(jPanelBottomControlLayout.createParallelGroup(Alignment.LEADING)
-                    .addGroup(jPanelBottomControlLayout.createSequentialGroup()
-                        .addComponent(RadioOneRowPerFile)
-                        .addGap(18, 18, 18)
-                        .addComponent(RadioOneRowPerIdentification)
-                        .addContainerGap(GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
                     .addGroup(Alignment.TRAILING, jPanelBottomControlLayout.createSequentialGroup()
                         .addGroup(jPanelBottomControlLayout.createParallelGroup(Alignment.TRAILING)
                             .addGroup(jPanelBottomControlLayout.createSequentialGroup()
                                 .addComponent(jLabel1)
                                 .addGap(18, 18, 18)
                                 .addComponent(cmdEncoding, GroupLayout.PREFERRED_SIZE, GroupLayout.DEFAULT_SIZE, GroupLayout.PREFERRED_SIZE)
-                                .addPreferredGap(ComponentPlacement.RELATED, GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                                .addPreferredGap(ComponentPlacement.RELATED, 586, Short.MAX_VALUE)
```

Comment:
These are the kind of values that NetBeans changes and I had tended to reset them back to how they used to be (without knowing too many details about them) 😄